GesturalScore.h

```diff
@@ -177,7 +177,7 @@ class GesturalScore : public TubeSequence
   
   void clear();
   void initTestScore();
-  void createFromSegmentSequence(SegmentSequence *origSegmentSequence);
+  void createFromSegmentSequence(SegmentSequence *origSegmentSequence, int enableConsoleOutput);
```

Comment:
Why is `enableConsoleOutput` an `int` and not a `bool`ean? Also, it should have a default value of `true` to ensure backwards compatibility.

Reply:
enableConsoleOutput is an int instead of boolean to be consistent with the other functions, e.g. vtlGesturalScoreToAudio, which were written by Peter. No other API function has a default value either, so adding this would be inconsistent. Further, the backend-dev is not backwards compatible to VTL2.3 at all.

Reply:
>enableConsoleOutput is an int instead of boolean to be consistent with the other functions, e.g. vtlGesturalScoreToAudio, which were written by Peter.
Point taken and accepted.
>No other API function has a default value either, so adding this would be inconsistent. 
You mean no other function in the public interface. That is necessary because the public interface must be C compliant, which does not support default arguments (no name mangling). This should not keep us from using default values in the non-public section of the code, which is free to use C++-only language features.
>Further, the backend-dev is not backwards compatible to VTL2.3 at all.
Breaking backwards compatibility in a place where it is unavoidable for progress is not a good reason to break it in another place where it is avoidable. Adding a default value here reduces the necessary changes to port outdated code to the most recent version.

Reply:
Added default value,
changed enableConsoleOutput in all API functions from int to bool, (asthey would have been casted to bool anyway in the other functions).

Reply:
Thanks.

Fun fact: `bool` was not even part of C before the C99 language standard and requires an additional header (`stdbool.h`) in a C compiler. Since we are compiling with a C++ compiler, I don't think this will be an issue. But it may explain why Peter chose to go with `int`s in the first place.

Reply:
I actually explicitly added the #include stdbool.h in the VocalTractLabApi.h, as it was indeed an issue for other compilers